core/Archive/ArchiveInvalidator.php

```diff
@@ -559,6 +559,7 @@ public function scheduleReArchiving($idSites, string $pluginName = null, $report
                 'segment' => $segment ? $segment->getString() : null,
             ]));
         } catch (\Throwable $ex) {
+            throw new $ex;
```

Comment:
Was that added for debugging only? If not it should be moved after the logging I guess.
Also shouldn't that simply be `throw $ex`? The `new` shouldn't be needed

Reply:
Yes debugging, will remove.